R/aaa-shared.R

```diff
@@ -0,0 +1,12 @@
+#' Parameters used in multiple functions
+#'
+#' Reused parameter definitions are gathered here for easier usage. Edit the
+#' definitions in `man/glossary/{term}.Rmd` (one file per term).
+#'
+#' @param dfMetrics
+#' ```{r, child = "man/glossary/dfMetrics.Rmd"}
```

Comment:
alternatively, we could just include this type of snippet in headers that have non-standard param configurations, so the description is still centralized.